Source/WebCore/html/HTMLElement.h

```diff
@@ -66,6 +67,8 @@ class HTMLElement : public StyledElement {
 
     WEBCORE_EXPORT String title() const final;
 
+    WEBCORE_EXPORT std::variant<String, RefPtr<TrustedScript>> innerTextForBinding();
+    WEBCORE_EXPORT virtual ExceptionOr<void> setInnerTextForBinding(const std::variant<String, RefPtr<TrustedScript>>&);
```

Comment:
Can we use two overloads here instead of `std::variant`? Seems like it would be higher performance and easier to read the code.